pkg/aws/eni/limits/limits_test.go

```diff
@@ -65,31 +58,35 @@ func (e *ENILimitsSuite) TestParseLimitString(c *check.C) {
 	limitString6 := ""
 
 	limit, err := parseLimitString(limitString1)
-	c.Assert(err, check.Equals, nil)
-	c.Assert(limit.Adapters, check.Equals, 4)
-	c.Assert(limit.IPv4, check.Equals, 5)
-	c.Assert(limit.IPv6, check.Equals, 6)
+	require.NoError(t, err)
+	require.Equal(t, 4, limit.Adapters)
+	require.Equal(t, 5, limit.IPv4)
+	require.Equal(t, 6, limit.IPv6)
 
 	limit, err = parseLimitString(limitString2)
-	c.Assert(err, check.Not(check.Equals), nil)
+	require.Error(t, err)
 
 	limit, err = parseLimitString(limitString3)
-	c.Assert(err.Error(), check.Equals, "invalid limit value")
-	c.Assert(limit.Adapters, check.Not(check.Equals), 4)
-	c.Assert(limit.IPv4, check.Not(check.Equals), 5)
-	c.Assert(limit.IPv6, check.Equals, 0)
+	require.Error(t, err)
+	require.Equal(t, "invalid limit value", err.Error())
+	require.NotEqual(t, limit.Adapters, 4)
+	require.NotEqual(t, limit.IPv4, 5)
+	require.Equal(t, limit.IPv6, 0)
 
 	limit, err = parseLimitString(limitString4)
-	c.Assert(err.Error(), check.Equals, "invalid limit value")
+	require.Error(t, err)
+	require.Equal(t, "invalid limit value", err.Error())
```

Comment:
nit: You can use `assert.ErrorContains` here to combine both of these checks:
```suggestion
	require.ErrorContains(t, err, "invalid limit value")
```

Reply:
thanks :+1: